libraries/helpers.rb

```diff
@@ -160,11 +160,11 @@ def port_open?(ip, port)
       false
     end
 
-    def data_dir(_version = node.run_state['mariadb']['version'])
+    def data_dir
```

Comment:
Why was this removed? It seems to be causing issues with unit tests.

Reply:
Explained in 4eb36e9eb8b0c562922d38ba3acd54a110c14204 and unit tests fixed in bb4f1724baad27d4dc531f082f44144782b9c14d